mappings/net/minecraft/enchantment/ChannelingEnchant.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_3587 net/minecraft/enchantment/ChannelingEnchant
```

Comment:
Should be renamed to `ChannelingEnchantment` for consistency with the other enchantments.

```suggestion
CLASS net/minecraft/class_3587 net/minecraft/enchantment/ChannelingEnchantment
```